sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -131,24 +131,25 @@ public async Task StartRecognizeContentPopulatesFormPage(bool useStream)
         public async Task StartRecognizeReceiptsPopulatesExtractedReceipt(bool useStream)
         {
             var client = CreateInstrumentedClient();
-            Operation<IReadOnlyList<RecognizedReceipt>> operation;
+            IReadOnlyList<RecognizedReceipt> receipts;
 
             if (useStream)
             {
                 using var stream = new FileStream(TestEnvironment.ReceiptPath, FileMode.Open);
-                operation = await client.StartRecognizeReceiptsAsync(stream, ContentType.Jpeg);
+                var operation = await client.StartRecognizeReceiptsAsync(stream, ContentType.Jpeg);
+
+                await operation.WaitForCompletionAsync();
+
+                Assert.IsTrue(operation.HasValue);
+                receipts = operation.Value;
             }
             else
             {
                 var uri = new Uri(TestEnvironment.ReceiptUri);
-                operation = await client.StartRecognizeReceiptsFromUriAsync(uri, default);
+                receipts = (await client.RecognizeReceiptsFromUriAsync(uri, default)).Value;
```

Comment:
I'm not wild about the need to put parentheses around the expression and have .Value at the end.

Is there another approach?  Krzysztof was ok with [this approach](https://github.com/Azure/azure-sdk-for-net-pr/pull/981/files#r399869464)...

Reply:
Implicit conversion to interfaces isn't allowed. Since `RecognizeReceiptsOperation` uses interface as its value type:
```
RecognizeReceiptsOperation : Operation<IReadOnlyList<RecognizedReceipt>>
```
we have to either return `IReadOnlyList<RecognizedReceipt>` from extension method instead of `Response<IReadOnlyList<RecognizedReceipt>>` or change value type to `ReadOnlyCollection` or some other concrete type.

Reply:
When writing TA samples, Krzysztof raised this concern too, and after looking at alternatives, we went with `.Value`